shared/test-esm-exports/README.md

```diff
@@ -14,21 +14,21 @@ So, `import Hls from 'hls.js'` would properly resolve `hls.js` and not fail tryi
 Working with workspaces and web-test-runner, to be able to point at this local package's dist, we need to make the working directory of our test runner be the root of the repo, rather than package that is being tested directly.
 
 In the `test` script, we need to add `--root-dir ../..` to `web-test-runner`, which points at the root of the repo.
-For mux-player, the test command could look like this:
+For ix-player, the test command could look like this:
```

Comment:
are we using this web-test-runner?

Reply:
comment: yes, 

note: run `yarn test` from root or package directory. This runs tests use `web-test-runner`.
```json
"test": "web-test-runner **/*test.js --port 8001 --coverage --config test/web-test-runner.config.mjs --root-dir ../..",
```